x-pack/plugins/security/server/plugin.ts

```diff
@@ -85,6 +86,10 @@ export interface SecurityPluginSetup {
    * Exposes services for audit logging.
    */
   audit: AuditServiceSetup;
+  /**
+   * Exposes services to access kibana roles per feature id with the GetDeprecationsContext
+   */
+  privilegeDeprecationServices: PrivilegeDeprecationsServices;
```

Comment:
Nit: this identifier is inconsistent throughout the PR ("privilege deprecation/s service/s").

Can we change it everywhere to `privilegeDeprecationsService` / `PrivilegeDeprecationsService`? That naming convention is consistent with the existing "DeprecationsService" that is provided by Core.